app/views/home/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<!-- Main content container -->
+<div class="main">
+    
+    <!-- Footer section -->
+    <footer>
+        
+        <!-- Link to navigate to the login page -->
+        <%= link_to "LOGIN", new_user_session_path, id: "login" %>
+        
+        <!-- Link to navigate to the sign-up page -->
+        <%= link_to "SIGN UP", new_user_registration_path, id: "signup" %>
+    </footer>
```

Comment:
- [ ] Please center the buttons and add your app logo on the top to match the given design: 

yours|given
---|---
![image](https://github.com/MohammadYaser/budget-app/assets/91486791/0ea3a6ea-ae3b-489e-9dca-f918a23b1740)|![image](https://github.com/MohammadYaser/budget-app/assets/91486791/d21cbaee-2281-4516-89ba-23df2955964d)